server/client.go

```diff
@@ -2249,19 +2249,18 @@ func (c *client) processSubsOnConfigReload(awcsti map[string]struct{}) {
 	// Collect client's subs under the lock
 	for _, sub := range c.subs {
 		subs = append(subs, sub)
```

Comment:
I would not add blindly here. Would do it only in a `else` of `if !canSubscribe()` and maybe only if `checkAcc` is true. Then the `for _, sub := range subs {` will be only for that. You would have to clean the comment before that loop though.

Reply:
ok will take a look.